code/Attack.ipynb

```diff
@@ -0,0 +1,1620 @@
+{
```

Comment:
<p>Consider including:</p><p> </p><ul><li>Code comments.</li><li>What the preprocessing steps are. </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sayakpaul/robustness-foundation-models/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>